### PR TITLE
[android][core] Track editing bug fix

### LINKED ISSUE #9326: Bookmarks editing is broken in master.
- Edit color - crash
- Change category - does nothing

Looks like https://github.com/organicmaps/organicmaps/pull/9033 breaks.

@rtsisyk @kavikhalique

### PR DESCRIPTION
**Pull request by kavikhalique**
Closes #9326 #9323

This PR contains some commits changes which got lost during merging of PR (https://github.com/organicmaps/organicmaps/pull/9033?notification_referrer_id=NT_kwDOBzKCIrUxMjAyNzc3ODczMzoxMjA3NTA2MjY#event-14329660536) into master

### CODE REVIEW

File: map/track.hpp [hunk 1]
```diff
@@ -55,6 +55,7 @@ class Track : public df::UserLineMark
   size_t GetLayerCount() const override;
   dp::Color GetColor(size_t layerIndex) const override;
   void SetColor(dp::Color color);
+  void SetColor(uint32_t color);
```
Comment (vng) [inline map/track.hpp:58]: Do we need this function if dp::Color is constructible from uint32_t rgba?

File: map/bookmark_manager.cpp [hunk 2]
```diff
@@ -2316,17 +2316,18 @@ void BookmarkManager::UpdateBookmark(kml::MarkId bmID, kml::BookmarkData const &
     SetLastEditedBmColor(bookmark->GetColor());
 }
 
-void BookmarkManager::UpdateTrack(kml::TrackId trackId, kml::TrackData const & trackData)
+void BookmarkManager::ChangeTrackColor(kml::TrackId trackId, uint32_t color)
```
Comment (vng) [inline map/bookmark_manager.cpp:2319]: I'd prefer to use dp::Color here and below, or name var as uint32_t rgba.

File: map/track.hpp (hunk 1 above)
Comment (kavikhalique) [reply map/track.hpp:58]: Oh yes, sorry i missed that, thanks will modify it

**Review by vng — APPROVED** [review]

File: data/strings/strings.txt [hunk 3]
```diff
@@ -31674,3 +31627,11 @@
...
     zh-Hans = 由于您的设备上未安装合适的应用程序，因此无法显示文件夹选择对话框。请安装文件管理器应用程序并重试。
     zh-Hant = 無法顯示資料夾選擇對話框，因為您的裝置上沒有安裝合適的應用程式。請安裝檔案總管應用程式並重試
+
+  [choose_color]
+    tags = android
+    en = Choose Color
+
+  [edit_track]
+    tags = android
+    en = Edit Track
```
Comment (pastk) [inline data/strings/strings.txt:31637]: @kavikhalique could you please open a PR with translations of "Edit Track" and "Choose Color" to other languages?

Comment (kavikhalique) [reply data/strings/strings.txt:31637]: Yeah sure i will do it shortly

**Pull request merged**